apps/page/option/import.ts

```diff
@@ -50,14 +50,12 @@ async function importClicked(): Promise<void> {
   /**
    * @type {HTMLTextAreaElement}
    */
-  const textArea = document.getElementById(
-    "importTextArea",
-  ) as HTMLTextAreaElement;
-  const text = textArea.value;
-  const lines = text.split("\n").filter((line) => line !== "");
-
-  const blockList = lines
-    .map((line) => {
+  const textArea = document.getElementById("importTextArea");
+  if (textArea instanceof HTMLTextAreaElement) {
+    const text = textArea.value;
+    const lines = text.split("\n").filter((line) => line !== "");
+
+    const blockList = lines.map((line) => {
```

Comment:
@vibh1103 

Please check for the existence of textArea immediately and perform an early return.
This will eliminate the need for indentation changes, making the diff easier to read.
The modifications needed for this block should be as follows, and it's likely that no further changes will be needed elsewhere.

```suggestion
  const textArea = document.getElementById("importTextArea");
  if (!(textArea instanceof HTMLTextAreaElement)) {
    throw new Error("importTextArea is not HTMLTextAreaElement");
  }

  const text = textArea.value;
  const lines = text.split("\n").filter((line) => line !== "");

  const blockList = lines.map((line) => {
```

Reply:
Done